tests/test_run_ccoverage.py

```diff
@@ -20,9 +24,22 @@
 
 class RunCcoverageTests(unittest.TestCase):
     """"TestCase class for functions in run_ccoverage.py"""
-    @pytest.mark.skip(reason="disable for now until actual use")
-    def test_main(self):  # pylint: disable=no-self-use
+
+    @staticmethod
+    @pytest.mark.skipif(distro.linux_distribution()[0] == "Ubuntu" and
+                        parse_version(distro.linux_distribution()[1]) < parse_version("16.04"),
+                        reason="Code coverage binary crashes in 14.04 Trusty but works in 16.04 Xenial and up")
+    @pytest.mark.slow
+    def test_main():
         """Run run_ccoverage with test parameters."""
         build_url = "https://build.fuzzing.mozilla.org/builds/jsshell-mc-64-opt-gcov.zip"
-        # run_ccoverage's main method does not actually return anything.
-        assert not run_ccoverage.main(argparse_args=["--url", build_url])
+
+        monkey = MonkeyPatch()
+        with monkey.context() as monkey_context:
+            monkey_context.setattr(gatherer, "RUN_COV_TIME", 3)
+            monkey_context.setattr("funfuzz.ccoverage.reporter.report_coverage",
```

Comment:
You could also ensure that `report_coverage` and `disable_pool` are called as you expect, instead of just logging. I usually do this by declaring a `dict()` in the method and then setting keys to assert later. Eg.

    calls = {"disable_pool": 0}
    monkeypatch.setattr("disable_pool", lambda: calls["disable_pool"] += 1)
    run_ccoverage.main()
    assert calls["disable_pool"] == 1

Reply:
We can't do an increment assignment in a lambda, can we? `lambda: calls["disable_pool"] += 1` seems to throw at `+=`.

Reply:
You're right, my bad. I am doing this somewhere, but I must have been using a non-lambda function, and I can't find it.